### DISCUSSION TITLE
Display Geometry in Avalonia 11

### DISCUSSION THREAD

**Discussion by DasLixou** [AvaloniaUI/Avalonia#11707] [Q&A] [answered]
Hello!
In Avalonia 10 i displayed SVGs by having them in a GeometryDrawing and then having this set as Drawing for a DrawingPresenter.
But in Avalonia 11, this isn't possible anymore. 
What is the new solution? And is it simpler?

Thanks :)

**Comment by kekekeks** [comment, marked as answer]
`Image` control with `DrawingImage`?

**Reply by ApaulMyLittleAirport** [reply]
please......GeometryDrawing does not have name and class attributes, how can I dynamically control the color of GeometryDrawing?

**Reply by DasLixou** [reply]
> please......GeometryDrawing does not have name and class attributes, how can I dynamically control the color of GeometryDrawing?

You can use Brush attribute on GeometryDrawing

**Comment by timothyparez** [comment]
Answering a question with a question is kind of... 

For those who run into this issue like I have, here's a proper example:

If you had something like this in v0.10:

```
<DrawingPresenter Drawing="{DynamicResource VSImageLib.GetHistoryListId_16x}"/>
```

You can replace it with this for v11:

```
<Image>
   <DrawingImage Drawing="{DynamicResource VSImageLib.GetHistoryListId_16x}"/>
</Image>
```

The resource is defined in `App.axaml`:

```
<Application.Resources>
    <DrawingGroup x:Key="VSImageLib.GetHistoryListId_16x"  >
        <GeometryDrawing Brush="#00FFFFFF" Geometry="F1M16,16L0.... " />
    </DrawingGroup>
</Application.Resources>
```
